metadata-integration/java/spark-lineage/src/main/java/datahub/spark/DatasetExtractor.java

```diff
@@ -149,8 +144,26 @@ private static Path getDirectoryPath(Path p, Configuration hadoopConf) {
         return p;
       }
     } catch (IOException e) {
-      // log.warn("Unable to get file system for path ", e);
       return p;
     }
   }
+
+  private static FabricType getCommonFabricType(Config datahubConfig) {
+    String fabricTypeString = datahubConfig.hasPath(DATASET_ENV_KEY)
+        ? datahubConfig.getString(DATASET_ENV_KEY).toUpperCase()
+        : "PROD";
+    FabricType fabricType = null;
+    try {
+      fabricType = FabricType.valueOf(fabricTypeString);
+    } catch (IllegalArgumentException e) {
+      Log.warn("Invalid env ({}). Setting env to default PROD", DATASET_ENV_KEY);
```

Comment:
parameter here should be `fabricTypeString` not `DATASET_ENV_KEY`

Reply:
Done